gatsby-config.js

```diff
@@ -117,6 +117,20 @@ module.exports = {
             },
           },
         },
+        tessen: {
+          product: 'DOC',
+          subproduct: 'TDOC',
+          segmentWriteKey: 'AEfP8c1VSuFxhMdk3jYFQrYQV9sHbUXx',
+          trackPageViews: true,
+          pageView: {
+            name: 'pageView',
+            category: 'DocPageView',
+            getProperties: ({ location, env }) => ({
+              location,
```

Comment:
2 things with this one:

* It looks like we call this `path` in the current docs site. Assuming we want to maintain consistency, we should probably do the same
* This is the `location` object for the page. We likely just want the `pathname` property off the location. Could you update to use `pathname`?

Reply:
```suggestion
              path: location.pathname,
```

Reply:
Oh gotcha, I was unclear what `location` represented. Good to know...I'll update.